module/applications/actor/character-sheet.mjs

```diff
@@ -214,7 +214,7 @@ export default class ActorSheet5eCharacter extends ActorSheet5e {
    * @returns {Promise}         Dialog or roll result.
    * @private
    */
-  _onSheetAction(event) {
+  async _onSheetAction(event) {
```

Comment:
This must be asynchronous to allow the promise used in in [lines 230 - 234](https://github.com/foundryvtt/dnd5e/blob/6998d9bbbb5f722f401851bfb2b75a563227e057/module/applications/actor/character-sheet.mjs#L230-L234) to function.